gradle/root/coverage.gradle

```diff
@@ -1,15 +1,27 @@
-apply plugin: 'jacoco'
+//TODO: Standard JaCoCo coverage doesn't work in Android module
+task mockitoCoverage(type: JacocoReport) {
```

Comment:
It looks somewhat confusing that "allprojects" section is within the body of the "mockitoCoverage" task configuration.

Reply:
It's caused by the fact that the root project in Mockito is not an empty parent project, but mockito-core in fact (which also generates coverage data).